src/pages/Home.tsx

```
@@ -1,15 +1,15 @@
 import React, { useEffect, useState } from 'react';
 import GlobalContainer from '../container/GlobalContainer';
-import Navbar from '../components/common/navbar';
-import { AnalyzingStatusCard } from '../components/home/AnalyzingStatusCard';
-import TitleText from '../components/common/titleText';
+import { Navbar, NavbarContainer } from '../components/common/navbar';
```

ESLint

[error] 'Navbar' is defined but never used.

(@typescript-eslint/no-unused-vars)